app/models/group.rb

```diff
@@ -414,6 +414,16 @@ def bulk_add(user_ids)
       if self.title.present?
         User.where(id: user_ids).update_all(title: self.title)
       end
+
+      if self.grant_trust_level.present?
+        User.where(id: user_ids).find_each do |user|
+          if user.trust_level < self.grant_trust_level
+            user.change_trust_level!(self.grant_trust_level)
+            user.trust_level_locked = true
+            user.save
```

Comment:
Unless we're checking for the return value, a good practice is to use `save!` or `update!`. Otherwise, this can fail silently without us know about it.